src/libPersistence/BlockStorage.cpp

```diff
@@ -675,8 +675,10 @@ bool BlockStorage::PutEpochFin(const uint64_t& epochNum) {
       DataConversion::StringToCharArray(to_string(epochNum)));
 }
 
-bool BlockStorage::GetMetadata(MetaType type, bytes& data) {
-  LOG_MARKER();
+bool BlockStorage::GetMetadata(MetaType type, bytes& data, bool muteLog) {
+  if (!muteLog) {
```

Comment:
Why need muteLog? g3log should be thread safe

Reply:
It's not due to safety issue. If we don't mute it for GetEpochFin specifically, the log will be flooded with this this.